src/Command/SshCommand.php

```diff
@@ -69,7 +69,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
 
             if (count($hostsAliases) === 1) {
-                $host = current($this->deployer->hosts->all());
+                $host = $this->deployer->hosts->get($hostsAliases[0]);
```

Comment:
I have two hosts:
- host 0: `Localhost`
- host 1: remote `Host`

The Localhost is removed from `$hostAliases`. But in this line it was chosen anyway.